tensorflow-ops/src/TensorFlow/Ops.hs

```diff
@@ -256,6 +257,15 @@ constant (Shape shape') values
                       [def & TensorShape.size .~ x | x <- shape']
                 & tensorVal .~ values
 
+-- | Reshape a N-D tensor down to a scalar.
+toScalar :: (TensorType a) 
+         => Tensor v a 
+         -> Tensor Value a
+toScalar = flip CoreOps.reshape (vector scalarShape)
+    where
+        scalarShape = [] :: [Int32]
```

Comment:
Maybe link to [reshape documentation](https://www.tensorflow.org/versions/r0.11/api_docs/python/array_ops.html#reshape)
which has an example?